css/styles.css

```diff
@@ -0,0 +1,839 @@
+  height: 96px;
+  gap: 24px;
+  margin: auto;
+  border: 4px;
+}
+
+.title {
+  text-align: center;
+  height: 24px;
+  font-weight: 600;
+  margin-bottom: 15px;
+  font-size: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.uld-tags {
+  /* width: 89px; */
+  display: flex;
+  gap: 8px;
+
+  /* margin-top: 16px; */
+  width: 271px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  padding-left: 2px;
+  margin: auto;
+}
+
+.btn-tag {
+  background: #fff;
+  border-radius: 4px;
+  border: 0;
+  width: 89;
+  height: 28px;
+  padding: 10px;
+}
+
+.li-tag-list {
+  list-style: none;
+  height: 30px;
+  color: #36b37e;
+  font-weight: 600;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form {
+  /* border: 2px solid skyblue; */
+  max-width: 1171px;
+  margin: auto;
+  display: flex;
+  padding: 80px 0;
+  gap: 24px;
+}
+
+@media (min-width: 768px) {
+  .form {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+@media (max-width: 768px) {
+  .form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+@media (min-width: 768px) {
+  .info {
+    flex: 0 1 40%;
+  }
+}
+
+.info p {
+  display: flex;
+  align-items: center;
+  text-align: justify;
+  max-width: 460px;
+  height: 256px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 52px;
+  color: #172b4d;
+  gap: 24px;
+}
+
+.form-details {
+  width: 100%;
+  max-width: 684px;
+  height: 330px;
+  display: flex;
+  gap: 24px;
+}
+
+@media (min-width: 768px) {
+  .form-details {
+    flex: 0 1 58%;
+  }
+}
+
+.formd {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  justify-content: center;
+  align-items: center;
+  text-align: justify;
+  padding: 10px 0;
+  margin: 10px 0;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+@media (max-width: 768px) {
+  .formd {
+    text-align: center;
+  }
+}
+
+.formd input {
+  width: 330px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  padding: 12px;
+}
+
+.formd .name {
+  width: 100%;
+  max-width: 684px;
+  margin: 10px 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  padding: 12px;
+}
+
+.formd .email {
+  width: 100%;
+  max-width: 684px;
+  margin: 10px 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  padding: 12px;
+}
+
+textarea {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  margin: 10px 0;
+  width: 100%;
+  max-width: 684px;
+  height: 114px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  padding: 12px;
+}
+
+.btn-primary {
+  width: 129px;
+  height: 48px;
+  gap: 10px;
+  background: #36b37f;
+  color: #fff;
+  border-radius: 8px;
+  border: 0;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+}
```

Comment:
- [ ] Please add styling for the other states of the buttons (`active` and `hover`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design: 😁

![image](https://user-images.githubusercontent.com/20567503/178081342-9170b3ca-bd87-4f4e-8465-4c41d6f0490d.png)

Reply:
Resolved